src/views/movie_list_view.js

```diff
@@ -0,0 +1,69 @@
+import Backbone from 'backbone';
+import MovieView from './movie_view';
+import Movie from '../models/movie';
+
+const MovieListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    this.collection = params.collection;
+
+    this.listenTo(this.model, 'update', this.render);
+    this.listenTo(this.collection, 'update', this.render);
+
+    this.mode = "inventory";
+  },
+
+  inventoryMode() {
+    this.mode = "inventory";
+  },
+
+  searchMode() {
+    this.mode = "search";
+  },
+
+  events: {
+    "click li.movie" : "toggleHighlight"
+  },
+
+  toggleHighlight(event) {
+    let active = event.currentTarget;
+    this.$el.find('li.movie').attr('class', 'movie highlighted');
+    this.$el.find('li.movie').not(active).attr('class', 'movie');
+  },
+
+  render() {
+    this.$('ul').empty();
+
+    if (this.mode == "search") {
+      this.collection.forEach((movie) => {
+        const movieView = new MovieView({
```

Comment:
We may have talked about this in person, but I'm not a big fan of the names `model` and `collection` for these instance variables. It would be very easy to get confused about which one is the rental library and which ones are movieDB results. Could you give them more descriptive names?